contributing/getting-started-quick.md

```diff
@@ -0,0 +1,77 @@
+
+* Create your local repository: `git clone git@github.com:<your username>/amphtml.git`
+* Add an alias: `git remote add upstream git@github.com:ampproject/amphtml.git`
+
+# Branch (do this each time you want a new branch)
+
+* Create the branch: `git branch --track <branch name> origin/master`
+* Go to the branch: `git checkout <branch name>`
+
+# Build AMP & run a local server
+* Make sure you have the latest packages (after you pull): `yarn`
+* Start the server: `gulp`
+* Access your server at [http://localhost:8000](http://localhost:8000)
+* Access your sample pages at [http://localhost:8000/examples](http://localhost:8000/examples)
+* Change the suffix for the examples from .html to .max.html to use your local JavaScript
+
+# Test AMP
+* Run the tests: `gulp test`
+* Run the tests in a specified set of files: `gulp test --files=<filename>`
+* Add the `--watch` flag to any `gulp test` command to automatically re-run the tests when a file changes
+* To run only a certain set of Mocha tests change  `describe` to `describe.only` for the tests you want to run; combine this with `gulp test --watch` to automatically rerun your test when files are changed   (but make sure to run all the tests before sending your change for review)
+
+# Create commits to contain your changes
+
+* Edit files in your favorite editor
+* Add each file you change: `git add <file>`
+* Create a commit: `git commit -m "<your commit message>"`
+* Instead of `add`ing each file individually you can use the `-a` flag on the commit instead
+
+# Pull the latest changes
+
+* `git checkout master`
+* `git pull upstream master`
+* `git checkout <branch name>`
+* `git rebase master`
+* Note that you may need to resolve conflicting changes at this point
+
+# Push your branch & create a Pull Request
+
+* Pull the latest changes as described above
+* `git checkout <branch name>`
+* `git push origin <branch name>`
+* Go to [https://github.com/ampproject/amphtml](https://github.com/ampproject/amphtml) and in the banner indicating you've recently pushed a branch, click the "Compare & pull request"  (if this banner does not appear, go to your fork at `https://github.com/<your username>/amphtml`, choose your branch from the "Branch" dropdown and click "New pull request")
+* Make sure you've signed the CLA (using the same email address as your git config indicates)
+* If your reviewer requests changes make them locally and then repeat the steps in this section to push the changes to your branch back up to GitHub again
+* If you don't get a new review within 2 business days, feel free to ping the pull request by adding a comment
+* Once approved your changes are merged into the amphtml repository by a core committer (you don't do this merge)
+
+# Delete your branch after your changes are merged (optional)
+
+* Go to the master branch: `git checkout master`
+* Delete your local branch: `git branch -D <branch name>`
+* Delete the GitHub fork branch: `git push origin --delete <branch name>`
```

Comment:
For follow up: Add information about AMP release cycle for when changes are expected to be available to all AMP pages!